firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -1134,9 +1136,11 @@ private void validateActiveTargets() {
       TargetData expectedTarget = expectedQueries.get(0);
       TargetData actualTarget = actualTargets.get(expected.getKey());
 
-      // TODO: validate the purpose of the target once it's possible to encode that in the
-      // spec tests. For now, only validate properties that can be validated.
+      // TODO: Simplify the TargetData validation when the sequenceNumber can be consistent as well.
       // assertEquals(expectedTarget, actualTarget);
+
+      assertEquals(expectedTarget.getPurpose(), actualTarget.getPurpose());
+
```

Comment:
ultranit: Can these blank line be removed, to put the check for getPurpose() right beside the other assertEquals() lines?